__tests__/get.test.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../db/connection"); //imports db again from connection
+const seed = require("../db/seeds/seed"); //imports seeding
+const request = require("supertest"); //imports supertest (a library which allows http express requests)
+const data = require("../db/data/test-data/index"); //imports test data
+const app = require("../app"); //imports app has all the application and routes in
+
+beforeEach(() => seed(data)); // sets up function to define what happens before each test, here we populate the database with data before testing
+afterAll(() => db.end()); //used to close connection after each test
+
+describe("GET /api/topics", () => {
+  test("should return an array of objects containing slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.topics.length).toBe(3); // checks topics should return array of 3
+        res.body.topics.forEach((topic) => {
+          // loop  through each topic in the res checks properties named slug and description
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+        });
+      });
+  });
```

Comment:
Really nice test here using forEach, good to see that you've checked the length of the body also. Consider using the following syntax here to make your test more robust by checking both the key and the value - 
'expect(topic).toHaveProperty('slug', expect.any(String))'

Reply:
Nice job adding this change but I think the intention was to replace lines 19 and 20 with lines like 21  so:
```
expect(topic).toHaveProperty("slug", expect.any(String));
expect(topic).toHaveProperty("topic", expect.any(String));
```
This would replace the original tests that only tested the key